data/json/npcs/exodii/exodii_Luliya.json

```
@@ -0,0 +1,239 @@
+[
+  {
+    "type": "npc",
+    "id": "exodii_materials",
+    "//": "Materials manager and music fan from another Ethiopia.",
+    "name_suffix": "Cyborg",
+    "class": "NC_EXODII_TYPE_2_Materials",
+    "name_unique": "Luliya",
+    "attitude": 0,
+    "mission": 7,
+    "chat": "TALK_LULIYA",
+    "faction": "exodii",
+    "traits": [ { "trait": "IGNORE_SOUND" }, { "trait": "NO_BASH" }, { "trait": "RETURN_TO_START_POS" } ]
+  },
+  {
+    "type": "npc_class",
+    "id": "NC_EXODII_TYPE_2_Materials",
+    "name": { "str": "Cyborg, type 2" },
+    "job_description": "I'm a cyborg.",
+    "traits": [ { "trait": "EXODII_BODY_2" } ],
+    "common": false,
+    "bonus_str": { "rng": [ 0, 6 ] },
+    "bonus_dex": { "rng": [ 0, 6 ] },
+    "bonus_int": { "rng": [ 0, 2 ] },
+    "bonus_per": { "rng": [ 0, 4 ] },
+    "carry_override": "NC_EXODII_TYPE_2_carried",
+    "worn_override": "EMPTY_GROUP",
+    "skills": [
+      {
+        "skill": "ALL",
+        "level": { "mul": [ { "one_in": 3 }, { "sum": [ { "dice": [ 2, 2 ] }, { "constant": 0 }, { "one_in": 4 } ] } ] }
+      },
+      { "skill": "mechanics", "bonus": { "one_in": 2 } },
+      { "skill": "electronics", "bonus": { "rng": [ 0, 2 ] } },
+      { "skill": "firstaid", "bonus": { "rng": [ 7, 10 ] } },
+      { "skill": "gun", "bonus": { "rng": [ 2, 4 ] } },
+      { "skill": "rifle", "bonus": { "rng": [ 2, 4 ] } },
+      { "skill": "unarmed", "bonus": { "rng": [ 2, 4 ] } },
+      { "skill": "speech", "bonus": { "rng": [ 5, 8 ] } }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_EXODII_TYPE_2_carried",
+    "//": "Golden krar for Luliya.",
+    "items": [ { "item": "krar_golden" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_LULIYA",
+    "dynamic_line": {
+      "u_has_var": "u_met_luliya",
+      "type": "general",
+      "context": "meeting",
+      "value": "yes",
+      "yes": [ "Inikwani dehina metahi tegwazhi." ],
+      "no": "[You see a tall golden figure in the closet with female features.  Strapped to the back of the head with copper wire is a glass cylinder containing long frizzy black hair.  The body stands still and emits an unpleasant beeping sound.  On the back there is a large red button with strange markings.  There is a golden harp of some kind.]"
```

```suggestion
      "no": "[You see a tall golden figure with female features.  Strapped to the back of the head with copper wire is a glass cylinder containing long frizzy black hair.  The body stands still and emits an unpleasant beeping sound.  On the back there is a large red button with strange markings.  There is a golden harp of some kind.]"
```
She's not in a closet, so this intro probably shouldn't introduce her as being inside a closet.